src/main/java/openfasttrack/importer/specobject/ImportHelper.java

```diff
@@ -105,46 +105,51 @@ private void foundStartElement(final StartElement element) throws XMLStreamExcep
         switch (element.getName().getLocalPart())
         {
         case SPECDOCUMENT_ROOT_ELEMENT_NAME:
-            LOG.finest("Found XML root element '" + SPECDOCUMENT_ROOT_ELEMENT_NAME + "' at "
+            LOG.finest(() -> "Found XML root element '" + SPECDOCUMENT_ROOT_ELEMENT_NAME + "' at "
```

Comment:
Is the old interface deprecated?
I found the other non-lambda version nicer to read.

And (without prove) I also suspect it to be faster.